### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by tdanford**
Includes the 'comparison' framework from adam-core, the genotype- and variant-level concordance code, and all the CLI commands that depend/run those classes, as well as test suites.

### CODE REVIEW

**Comment by fnothaft** [comment]
I've just set Jenkins up: [qc-metrics](https://amplab.cs.berkeley.edu/jenkins/job/qc-metrics/), [qc-metrics-prb](https://amplab.cs.berkeley.edu/jenkins/job/qc-metrics-prb/). Time to see if it works...

**Comment by fnothaft** [comment]
Jenkins, test this please.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+    <scala.artifact.suffix>2.10</scala.artifact.suffix>
+    <spark.version>1.2.0</spark.version>
+    <parquet.version>1.6.0rc4</parquet.version>
+    <avro.version>1.7.6</avro.version>
+    <!-- Edit the following line to configure the Hadoop (HDFS) version. -->
+    <hadoop.version>2.2.0</hadoop.version>
+    <scoverage.version>0.99.2</scoverage.version>
+
+    <adam.version>0.16.0</adam.version>
+    <utils.version>0.1.1</utils.version>
```
Comment (fnothaft) [inline pom.xml]: If you're on Spark 1.2, you should point at bdg-utils 0.1.2.

**Comment by fnothaft** [comment]
w00t! @AmplabJenkins is alive and well. @tdanford can you fix the one nit surrounding the bdg-utils version, and then I'll commit?

File: pom.xml (hunk 1 above)
Comment (tdanford) [inline pom.xml]: Done.

**Comment by tdanford** [comment]
@fnothaft I fixed the bug with the bdg-utils version, and the build passed.

**Comment by fnothaft** [comment]
Thanks @tdanford!

**Comment by tdanford** [comment]
Thanks @fnothaft !

**Pull request merged**